src/context/analytics.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useContext } from "react";
+import NewRelicAPI from "new-relic-browser";
+import { useQuery } from "@apollo/react-hooks";
+import { GetUserQuery } from "gql/generated/types";
+import { GET_USER } from "gql/queries";
+import get from "lodash/get";
+import { useLocation } from "react-router-dom";
+
+declare global {
+  interface Window {
+    newrelic: typeof NewRelicAPI; // eslint-disable-line
+  }
+}
+
+type SendEvent = (
+  eventName: string,
+  attributes?: { [key: string]: string | number }
+) => void;
+
+interface Analytics {
+  sendEvent: SendEvent;
+}
+
+const AnalyticsContext = React.createContext<Analytics | null>(null);
```

Comment:
we can avoid using a context and use a hook instead since this component isn't passing values that depend on state to it's children

Reply:
i think that this allows for more scalability should any of the requirements around analytics change